man/data.table.Rd

```diff
@@ -418,6 +419,9 @@ DT[, list(MySum=sum(v),
           MyMax=max(v)),
     by=.(x, y\%\%2)]                    # by 2 expressions
 
+DT[, .(f=f(a), m=g(a)), by=b, verbose=TRUE,
+    env=list(f="sum", g=mean, b="x")] # subtitution via env arg
```

Comment:
As Jan points out, using `"sum"` is to be strongly preferred. I would not document `g=mean` so prominently as that is not the "correct" way to use `env=`. Better to leave that approach unstated -- the discussion+related bugfix in #6026 is about using `g=mean` leading to unexpected behavior/archaic error. Better to steer users to the well-trodden path of using `f="sum"`.

Reply:
yeah i just thought that if we implemented a change, it might be good idea to document this as an alternative method, but yeah if `"sum"` is the correct way and `g= mean` is not recommended then its better to just remove it.